### PR TITLE
PR for #165: prepare release xtevent 3.0.0

### LINKED ISSUE #165: prepare release xtevent 3.0.0
In this issue, we'll update documentation for the release of `xtevent 3.0.0`

- [x]  Update home page
- [x]  Update version statements in ado files
- [x]  Check examples in home page
- [x]  Check examples in test file and run
- [x]  Check examples in help files and run
- [x]  Check that help files display properly
- [x]  Update acknowledgments
- [x]   Update install.txt and release.txt
- [x] Update version [dependencies ](https://github.com/JMSLab/xtevent/blob/main/dependency.do) in `dependency.do`

### PR DESCRIPTION
**Pull request by Constantino-Carreto-Romero**
Closes #165

### CODE REVIEW

**Comment by jorpppp** [comment]
@Constantino-Carreto-Romero I was reviewing the PR and found this issue with reghdfe: https://github.com/JMSLab/xtevent/issues/165#issuecomment-1894629559

I think it would be a good idea to switch `xtevent`'s default behavior to try to install the Github version of reghdfe first. If that doesn't work, issue a warning and install the ssc version. Do you think you can implement that change?

File: README.md [hunk 1]
```diff
@@ -7,12 +7,18 @@
 `xtevent` is a Stata package to estimate linear panel event-study models. It includes three commands: `xtevent` for estimation; `xteventplot` to create event-study plots and; `xteventtest` for post-estimation hypotheses testing. 
 
 
-- Last version: 2.2.0 (15mar2023)
+- Last version: 3.0.0 (jan2024)
```
Comment (jorpppp) [inline README.md:10]: @Constantino-Carreto-Romero we'll need to make one more push to add the day here.

File: test/test.do [hunk 2]
```diff
@@ -32,29 +32,29 @@ xtevent y eta, policyvar(z) panelvar(z) window(5)
 xtevent y eta, policyvar(z) timevar(z) window(5) 
 */
 
-* Testing noci, nosupt, nozeroline, nominus1label
+* Testing noci, nosupt, nozeroline, nonormlabel
```
Comment (jorpppp) [inline test/test.do:35]: @Constantino-Carreto-Romero We need to document this change in the changelog in README.md please

File: xtevent/get_unit_time_effects.ado [hunk 3]
```diff
@@ -1,4 +1,4 @@
-*! get_unit_time_effects.ado 2.2.0 March 15 2023
+*! get_unit_time_effects.ado 3.0.0 January, 2024
```
Comment (jorpppp) [inline xtevent/get_unit_time_effects.ado:1]: @Constantino-Carreto-Romero day pending

**Comment by jorpppp** [comment]
@Constantino-Carreto-Romero Have you had the chance to look at these pending changes?

File: README.md (hunk 1 above)
Comment (Constantino-Carreto-Romero) [reply README.md]: updated in https://github.com/JMSLab/xtevent/pull/168/commits/90d6dbc69456faae71e93088b07b8301d992c9f9

File: xtevent/get_unit_time_effects.ado (hunk 3 above)
Comment (Constantino-Carreto-Romero) [reply xtevent/get_unit_time_effects.ado]: updated release date in ado files and help files in https://github.com/JMSLab/xtevent/pull/168/commits/a7c402563f6235798efe08e3d477aff843e83446

File: test/test.do (hunk 2 above)
Comment (Constantino-Carreto-Romero) [reply test/test.do:35]: I documented it in the version's list of changes at https://github.com/JMSLab/xtevent/pull/168/commits/2fcc27a3f8de7fedb1ea255a269efd2b373631ad. I added that description to the draft of the release as well https://github.com/JMSLab/xtevent/releases.

Comment (Constantino-Carreto-Romero) [reply test/test.do:35]: More detailed description about the option's changed name in https://github.com/JMSLab/xtevent/pull/168/commits/1514c451d0793982cdd071b7feb1381986faef45. I also added it to the release' draft.

**Comment by Constantino-Carreto-Romero** [comment]
> @Constantino-Carreto-Romero I was reviewing the PR and found this issue with reghdfe: [#165 (comment)](https://github.com/JMSLab/xtevent/issues/165#issuecomment-1894629559)
> 
> I think it would be a good idea to switch `xtevent`'s default behavior to try to install the Github version of reghdfe first. If that doesn't work, issue a warning and install the ssc version. Do you think you can implement that change?

@jorpppp 
I checked again the problem I mentioned in https://github.com/JMSLab/xtevent/issues/165#issuecomment-1894629559. That time I found as solution to install `ivreghdfe` along with its dependency (`reghdfe`) using the latest GitHub versions (instead of the ssc versions). I revised this problem again and found that it is enough to only install the Github version of `ivreghdfe`, so `reghdfe` can be installed as the ssc version. By doing this, I run the test file without problems. 

> I think it would be a good idea to switch `xtevent`'s default behavior to try to install the Github version of reghdfe first. If that doesn't work, issue a warning and install the ssc version. Do you think you can implement that change?

At first, I implemented it as indicated, but since the dependency file is a do-file and not an ado-file, it always printed the warning's text. Then, I only added some commented text to the dependency file in https://github.com/JMSLab/xtevent/pull/168/commits/4a55e790872e6a3c65c480bd2a8fc7810c9e177c

**Comment by jorpppp** [comment]
@Constantino-Carreto-Romero thanks! This looks like it's ready to release!

**Comment by jmshapir** [comment]
@Constantino-Carreto-Romero @jorpppp fyi that the [release](https://github.com/JMSLab/xtevent/releases/tag/untagged-11a2b13f7db4f9eca66d) is still in draft status. It won't appear as most recent until it is published.

Thanks!

**Comment by jorpppp** [comment]
> @Constantino-Carreto-Romero @jorpppp fyi that the [release](https://github.com/JMSLab/xtevent/releases/tag/untagged-11a2b13f7db4f9eca66d) is still in draft status. It won't appear as most recent until it is published.
> 
> Thanks!

Thanks @jmshapir ! Your comment caught us in the middle of publishing the release. It should be published now. We pushed forward today to have a baseline for the upcoming changes.

**Pull request merged**